consensus/module.go

```diff
@@ -124,53 +129,20 @@ func Create(configPath, genesisPath string, useRandomPK bool) (modules.Consensus
 		validatorMap: valMap,
 
 		utilityContext:    nil,
-		paceMaker:         paceMaker,
-		leaderElectionMod: leaderElectionMod,
+		paceMaker:         pacemakerMod,
```

Comment:
You renamed the variable `pacemakerMod` but kept the structure field the legacy `pacemaker` let's be consistent with that